### ISSUE TITLE
dropbox backend does not overwrite existing files

### ISSUE THREAD

**Issue by epetxepe** [jschneier/django-storages#138]
This backend has no similar option to "file_overwrite = setting('AWS_S3_FILE_OVERWRITE', True)" as S3boto does.

I've overcome this issue by updating _save method with solution at http://stackoverflow.com/questions/31909627/dropbox-files-not-being-overwritten

```
def _save(self, name, content):
        #name = name.replace ('\\','/')     #137 
        file_metadata = self.client.metadata(name)
        the_rev = file_metadata['rev']
        self.client.put_file(name, content, overwrite=True, parent_rev=the_rev)
        return name
```

I think it would be interesting to add something similar for this backend

**Comment by jschneier** [comment]
Sure! Would you be interesting in creating a pull request for this functionality?

Also, from that Stackoverflow link it looks like dropbox will ignore the write if the content is the same. I don't think it's that important for now but something to keep in mind.

**Comment by riccardodivirgilio** [comment]
+1

**Comment by jschneier** [comment]
Fixed in #873.
